package.json

```diff
@@ -1,97 +1,48 @@
 {
-  "name": "@contentful/live-preview",
-  "version": "2.4.7",
+  "name": "@contentful/preview",
+  "private": true,
   "author": "Contentful GmbH",
   "license": "MIT",
-  "description": "Preview SDK for both the field tagging connection + live content updates",
-  "source": "./src/index.tsx",
-  "main": "./dist/index.cjs",
-  "module": "./dist/index.js",
-  "typings": "./dist/index.d.ts",
-  "type": "module",
-  "files": [
-    "dist",
-    "react",
-    "style.css"
-  ],
-  "exports": {
-    ".": {
-      "import": "./dist/index.js",
-      "require": "./dist/index.cjs",
-      "types": "./dist/index.d.ts"
-    },
-    "./react": {
-      "import": "./dist/react.js",
-      "require": "./dist/react.cjs",
-      "types": "./dist/react.d.ts"
-    },
-    "./style.css": {
-      "import": "./dist/style.css",
-      "require": "./dist/style.css"
-    }
-  },
+  "description": "all around preview content in contentful",
+  "homepage": "https://github.com/contentful/live-preview#readme",
   "repository": {
     "type": "git",
     "url": "git+https://github.com/contentful/live-preview.git"
   },
   "bugs": {
     "url": "https://github.com/contentful/live-preview/issues"
   },
+  "workspaces": [
+    "packages/*"
+  ],
   "scripts": {
-    "start": "vite build --watch",
-    "build": "vite build",
-    "lint": "eslint ./src --ext .js,.jsx,.ts,.tsx",
-    "tsc": "tsc --noEmit",
-    "test": "vitest",
-    "test:ci": "vitest run",
-    "coverage": "vitest run --coverage",
+    "lint": "lerna run lint",
+    "tsc": "lerna run tsc",
+    "test": "lerna run --stream test",
+    "test:ci": "lerna run test:ci",
+    "coverage": "lerna run coverage",
     "prepare": "husky install",
-    "presemantic-release": "yarn build",
+    "build": "lerna run build",
```

Comment:
we could add turborepo like we have in field editors & use it for the build command it makes it quite fast once we start to add more packages